### DISCUSSION TITLE
Example for using MaskLabel

### DISCUSSION THREAD

**Discussion by JiaruiWang** [pyg-team/pytorch_geometric#4773] [Q&A]
Hi,

I am trying to use MaskLabel in my project. I tried to convert  [unimp_arxiv.py](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/unimp_arxiv.py)  to an OGB Product dataset version that uses NeighborSampler. But I am having some difficulties.
May I ask for an example of an OGB product using NeighborSampler or some help?
I got an error for importing MaskLabel
```
from torch_geometric.nn import MaskLabel
ImportError: cannot import name 'MaskLabel' from 'torch_geometric.nn'
```

My PyG version is 2.0.4

Thank you

**Comment by rusty1s** [comment]
We just integrated this feature to PyG, so it is not available yet in version 2.0.4. You can follow [this guide](https://github.com/pyg-team/pytorch_geometric#nightly-and-master) on how to make use of newly integrated features right away.
